nodejs-runner/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mhart/alpine-node:base-10.0.0 AS release
```

Comment:
Used this 3rd-party base image to get rid of yarn and npm that are in the official node image by default: https://github.com/nodejs/docker-node/blob/2ecc9e8579f519ae3d267b5b497b8c04d6c7040d/10/alpine/Dockerfile#L58

Reply:
We're quite heavily relying on yarn. What are the reasons for removing it?

Reply:
This should act only as nodejs runtime on any environment (default `production`). So no need for yarn or npm here. The dependencies will be handled in another build stage.

Reply:
Makes sense, thanks for clearing this up 👍

Reply:
@jzaefferer @marvinhagemeister For better understanding of the concept I added 2 more commits:

https://github.com/sevenval/dockerfiles/pull/1/commits/0fbfa864cfdf742b2d85261110f5043de51daa12

and

https://github.com/sevenval/dockerfiles/pull/1/commits/668470c7885b60ee634e9e969ab529a4dd5bc080